src/user/dto/userInfo.dto.ts

```diff
@@ -8,7 +8,7 @@ export class UserInfo {
 
   @IsString()

```

Comment:
this will flag off when longitude and latitude is received from the frontend.... json objects as against string validator